joint_names:
  - panda_finger_joint1
  - panda_finger_joint2
